packages/story-editor/src/components/canvas/karma/cloneSelection.karma.js

```diff
@@ -22,103 +22,114 @@ import { useStory } from '../../../app/story';
 
 describe('Clone element integration', () => {
   let fixture;
-  let bg, img1, img2;
+  let bg, bgRect;
```

Comment:
@barklund See the Relevant Technical Choices section in the PR description that explain the extensive changes here.